src/routes/(site)/6/teams/members.svelte

```diff
@@ -0,0 +1,26 @@
+<script lang="ts">
+  import type { Team, Term } from '$lib/public/board';
+  import { OFFICERS_JSON } from '$lib/public/board/data';
+  import { getMembers } from '$lib/public/board';
+  import OfficerProfile from '../../about/officer-profile.svelte';
+
+  export let data: {
+    term: Term;
+    team: Team;
+  };
+
+  const members = getMembers(OFFICERS_JSON, data.term, data.team.tiers);
+</script>
+
+<section class="team">
+  <h2>{data.team.title}</h2>
+  <p>{data.team.description}</p>
+  <ul>
+    <!-- TODO: Fix each loop to show the correct officers per term. -->
```

Comment:
TODO(@EthanThatOneKid): Fix bug!

Reply:
Resolved in <https://github.com/EthanThatOneKid/acmcsuf.com/pull/955/commits/899530b1092bea858769adea8431667bcc988b31>.